src/campaign/sass/components/_button.scss

```diff
@@ -105,9 +105,9 @@ $offset: 2;
 }
 
 .button--sparks{
```

Comment:
the sparks on a button should only be above/below, they now also come out of the sides